daemon/daemon.go

```diff
@@ -15,6 +15,9 @@ import (
 	"time"
 
 	"github.com/docker/libcontainer/label"
+	"github.com/docker/libnetwork"
+	"github.com/docker/libnetwork/pkg/netlabel"
```

Comment:
packages in `pkg` are meant to be externally visible packages which can be consumed by anybody outside the context of libnetwork. Other packages in libnetwork are just internal packages to avoid putting all the libnetwork specific code in one directory because of the lack of any good way to put files of the same package in multiple directories